src/providers/LocalizationProvider.tsx

```diff
@@ -49,19 +49,6 @@ export default function LocalizationProvider({
     fetchTimeZones();
   }, [selectedLocale]);
 
-  useEffect(() => {
```

Comment:
Why was this removed?

Reply:
This was used to update the locale of the cookie-script pop-up.  Since we don't have that pop-up anymore we don't need to update it's locale.